Adds documentation for sync button in the analytics dashboards. (#915)
wso2/docs-apim: en/docs/learn/analytics/analyzing-apim-statistics-with-batch-analytics/viewing-api-statistics.md

</html>

The availability of APIs statistics is directly related to the [Availability of APIs (health monitoring)](../../../../learn/analytics/managing-alerts-with-real-time-analytics/alert-types/#availability-of-apis-api-health-monitoring) alert type.

## Date Range Picker

> This can be used to configure the **time period** and **granularity mode** of the data should be visualized or reported in the widgets listed above. Here the aggregated data to the selected granularity will be visualized or reported in widgets.

Reviewer: ```suggestion
This can be used to configure the **time period** and **granularity mode** of the data which should be visualized in the widgets listed above.
```

Author: Resolved with bea00a769294fb86757ff2bc3fb05557bffa47f2

The following diagram shows the configurable options in this widget.  

<html>
<body>
<img src="../../../../assets/img/learn/publisher-date-time-picker.png" width="500"/>
</body>
</html>

The time period can be defined, either choosing from `Quick Ranges` or defining `Custom Ranges`. For the quick ranges `1 Min`, `15 Min`, `1 Hour`, `1 Day`, `7 Days`, `1 Month`, `3 Months`, `6 Months`, and `1 Year` options are provided.
For `Custom Ranges` users can define the start date as `From` and end date as `To`.

For `Quick Ranges` the **granularity mode** selected as default and for `Custom Ranges`, users can choose from `second`, `minute`, `hour`, `day`, `month` and `year`.

> Generally the widgets will visualize or report the data retrieved during the initialization. But if a user required to update the widget data with time, can toggle the `sync button` at the upper right corner of the diagram.

Reviewer: ```suggestion
Generally, the widgets will visualize or report the data retrieved during the initialization. But if a user is required to update the widget with new data since the widget's initial load, they can toggle the `sync button` at the upper right corner of the diagram.
```

Reviewer: We also need to insert an image highlighting this button.

Author: Resolved with bea00a7

!!! note

>         During the `synced mode` the `To` value of the Date Range Picker updated to the current time for every 30 seconds and all the widgets will retrieve the data for updated time period.  

Reviewer: ```suggestion
        During the `synced mode` the `To` value of the Date Range Picker will be updated to the current timestamp every 30 seconds and all the widgets will retrieve the data for the updated time period.  This mode can be useful to present the dashboard in a non-interactive display.
```

Author: Resolved with bea00a7
